litellm/llms/vertex_ai/vertex_llm_base.py

```diff
@@ -99,7 +105,7 @@ def load_auth(
             if project_id is None:
                 project_id = creds_project_id
 
-        creds.refresh(Request())  # type: ignore
+        self.refresh_auth(creds)
```

Comment:
Switched to using this so I could mock it out in a test (the current test suite mocks out this entire method, but I want to actually test this thing)